source/administration/changelog.md

```diff
@@ -2,6 +2,224 @@
+
+ - Under `SamlSettings` in `config.json`:
+   - Added `"Enable": false` to allow login using SAML. See [documentation](http://docs.mattermost.com/deployment/sso-saml.html) to learn more about configuring SAML for Mattermost.
+   - Added `"Verify": false` to control whether Mattermost verifies the signature sent from the SAML Response matches the Service Provider Login URL.
+   - Added `"Encrypt": false`to control whether Mattermost will decrypt SAML Assertions encrypted with your Service Provider Public Certificate.
+   - Added `"IdpUrl": ""` to set the SAML SSO URL where Mattermost sends a SAML request to start login sequence.
+   - Added `"IdpDescriptorUrl": ""` to set the Identity Provider Issuer URL for the Identity Provider you use for SAML requests.
+   - Added `"AssertionConsumerServiceURL": ""` to set the Service Provider Login URL.
+   - Added `"IdpCertificateFile": ""` to set the public authentication certificate issued by your Identity Provider.
+   - Added `"PublicCertificateFile": ""` to set certificate used to generate the signature on a SAML request to the Identity Provider for a service provider initiated SAML login, when Mattermost is the Service Provider.
+   - Added `"PrivateKeyFile": ""` to set the private key used to decrypt SAML Assertions from the Identity Provider.
+   - Added `"FirstNameAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the first name of users in Mattermost.
+   - Added `"LastNameAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the last name of users in Mattermost.
+   - Added `"EmailAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the email of users in Mattermost.
+   - Added `"UsernameAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the username of users in Mattermost.
+   - Added `"NicknameAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the nickname of users in Mattermost.
+   - Added `"LocaleAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the language of users in Mattermost.
+   - Added `"LoginButtonText": ""` set the text that appears in the login button on the login page.
+
+#### Database Changes from v3.1 to v3.2
+
+**TeamMembers Table**    
+- Added `DeleteAt` column.
+
+**Emoji Table**    
+- Added `Emoji` table.
+
+### Known Issues
+
+- “More” option under Direct Message list no longer shows count of team members not in your direct message list.
+- On Firefox, search results for hashtags are not properly highlighted.
+- Clicking on a desktop notification from another team doesn’t open the team.
+- Webhook attachments don't show up in search results.
+- On Firefox, System Console sidebar completely disappears when an LDAP setting is saved
+- On Firefox, `CTRL/CMD + U` keyboard shortcut doesn't work
+- Copying and pasting an image from a browser doesn't work
```

Comment:
This was fixed, (for Chrome + Edge, and it is not a supported feature for other browsers so not really a bug).